app/src/main/res/layout/fragment_mobile_app_integration.xml

```diff
@@ -140,6 +140,35 @@
                 android:layout_marginEnd="50dp"
                 android:text="@string/pref_location_background_summary" />
 
+            <androidx.appcompat.widget.AppCompatTextView
+                style="@style/TextAppearance.HomeAssistant.Title"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/activity_margin"
+                android:text="@string/permission_explanation_calls" />
+
+            <androidx.appcompat.widget.AppCompatButton
+                android:id="@+id/phone_state_perms"
+                style="@style/Widget.HomeAssistant.Button.Colored"
```

Comment:
We may or may not need to remove this, can you test locally with both dark mode and light mode enabled in the system UI? If the button is missing in light mode just remove this line and it should correct it.

Reply:
Oh, i've got what you mean after repository update)